terminus/geometry/latlon.py

```diff
@@ -89,3 +96,44 @@ def delta_in_meters(self, other):
             return (delta_lat_in_meters, 0)
         else:
             return (delta_lat_in_meters, delta_lon_in_meters)
+
+    def midpoint(self, other):
+        '''
+        This method gives an approximation of the midpoint between two LatLon
+        objects, but it is not the 'real' midpoint. It works pretty well when the
```

Comment:
maybe worth explaining why is not the real midpoint? (just saying that we assume a plane instead of a sphere for point calculations will be more than enough)